src/components/Facets/Disjunctive/index.js

```diff
@@ -32,7 +32,8 @@ var DisjunctiveFacet = React.createClass( {
                 onClick={ this.toggleSelect.bind( this, facet.name, facetValue.name ) }>
                <input type="checkbox"
                       onClick={ function( e ) { e.preventDefault(); } }
-                      checked={ facetValue.isRefined } />
+                      checked={ facetValue.isRefined }
+                      readOnly/>
```

Comment:
Ok. So just to get the vocabulary straight: Any DOM element generated by React is called a component (not only the top level one). And as you're here setting the `checked` value through React, the checkbox is considered a controlled component (and thus needs either a `readonly` or an `onChange`).

Did I get that correctly ?